workflows4s-web-ui/build.sbt

```diff
@@ -0,0 +1,41 @@
+import scala.sys.process._
+import scala.language.postfixOps
+
+import sbtwelcome._
+
+Global / onChangedBuildSource := ReloadOnSourceChanges
+
+lazy val workflows4s-web-ui =
+  (project in file("."))
+    .enablePlugins(ScalaJSPlugin)
+    .settings( // Normal settings
+      name         := "workflows4s-web-ui",
+      version      := "0.0.1",
+      scalaVersion := "3.6.4",
+      organization := "myorg",
+      libraryDependencies ++= Seq(
+        "io.indigoengine" %%% "tyrian-io" % "0.14.0",
+        "org.scalameta"   %%% "munit"     % "1.1.1" % Test
+      ),
+      testFrameworks += new TestFramework("munit.Framework"),
+      scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) },
+      scalafixOnCompile := true,
+      semanticdbEnabled := true,
+      semanticdbVersion := scalafixSemanticdb.revision,
+      autoAPIMappings   := true
+    )
```

Comment:
Remove this file, and move this section (possibly minus lines 22-25) down to the main `build.sbt` file in the root of the repo, and change:

```scala
lazy val workflows4s-web-ui =
  (project in file("."))
```

to:

```scala
lazy val workflows4sWebUi =
  (project in file("workflows4s-web-ui"))
```